scripts/ci-packet-deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The purpose of the script is to deploy k8s cluster in the packet.net
+# PACKET_AUTH_TOKEN and PACKET_PROJECT_ID MUST be provided externally
+
+K8S_DEPLOYMENT_NAME=nsm-ci-"$CIRCLE_BUILD_NUM"
+
+docker run \
```

Comment:
Right, currently multinode functionality is not developed, well it is not even designed as we do not have yet multinode api. For now a single node cluster is the only working and tested scenario.

Reply:
Currently, we're using the default cross-cloud ci configuration, which is 3 master, 1 worker node set up.

Reply:
@mestery The other option would be to simply pin the NSE and NSC to the same node for the moment.  Likely simpler than hacking at the Cross Cloud CI scripts.

Reply:
@mestery Also... pinning the NSE and NSC to the same node would make it simpler when we start testing for multi-node, because then the Packet.net CI stuff doesn't have to change.  Should make things simpler for the folks implementing and testing the multi-node functionality as they don't have to understand how to change the Packet.net CI to get more than one worker node.

Another possibility might be to simply taint 2 of the three workers... lots of ways to approach the issue :)

Reply:
@mestery And I just read @platoff 's comments and felt silly at commenting on a current non-issue ;)